server.js

```diff
@@ -55,6 +55,12 @@ app.prepare().then(() => {
         Location: req.url.replace('commands-reference', 'command-reference')
       })
       res.end()
+    } else if (pathname == '/doc/tutorial') {
+      // path /doc/tutorial -> /doc/tutorials
+      res.writeHead(302, {
+        Location: req.url.replace('/doc/tutorial', '/doc/tutorials')
```

Comment:
Maybe this should use a regex to match exacly `/^\/doc\/tutorial/$` here
and separately add something like `/^\/doc\/tutorial(\/.*)` that redirects to `/\/doc\/tutorials\/tutorial\1/`.

> BTW `.../tutorials/tutorial/...`? will comment on this below...

Reply:
It seems simpler without a regex, why should we complicate it?

Reply:
@dashohoxha I think it's better indeed to handle a broader condition. At least to redirect `/doc/tutorial/` properly.